pkg/tfbridge/schema_test.go

```diff
@@ -2808,7 +2797,8 @@ func TestExtractSchemaInputsNestedMaxItemsOne(t *testing.T) {
 				},
 			},
 			expectedOutputs: resource.PropertyMap{
-				"id": resource.NewProperty("MyID"),
+				"id":     resource.NewProperty("MyID"),
+				"__meta": resource.NewStringProperty("{\"schema_version\":\"0\"}"),
```

Comment:
We can throw a ticket somewhere that the bridge shouldn't inject annoying `"__meta": resource.NewStringProperty("{\"schema_version\":\"0\"}"` that are equal to the default value.

Reply:
https://github.com/pulumi/pulumi-terraform-bridge/issues/2428